### PR TITLE
Fix scrolling detection on activity 14

### LINKED ISSUE #292: Filters don't work on 14
### Steps to reproduce

1. Click on "Von Ihnen"
2.
3.

### Expected behaviour

I should my activities only

### Actual behaviour

I see activities of other people.
There also seem to be a infinite loop somewhere which ends up crashing the tab.

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #292

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -51,23 +51,23 @@ $(function(){
...
-		$content: $('#app-content'),
+		$content: $(document),
 		firstKnownId: 0,
 		lastGivenId: 0,
 		activities: {},
 
 		prefill: function () {
 			this.ignoreScroll += 1;
-			if (this.$content.scrollTop() + this.$content.height() > this.$container.height() - 100) {
+			if (this.$content.scrollTop() + $(window).height() > this.$container.height() + 100) {
```
Comment (nickvergessen) [inline js/script.js]: I will make it -100 again, so new activities are already loaded, before you reach the end, so ideally you never see the loading circle

Comment (skjnldsv) [reply js/script.js]: Ah, make sense now. I couldn't figure why it was set to -100!

**Comment by nickvergessen** [comment]
Can you prepare a backport on stable14?

**Comment by skjnldsv** [comment]
Of course! :)

**Comment by MorrisJobke** [comment]
@skjnldsv 🏓

**Comment by MorrisJobke** [comment]
Ah it was already backported in #295

**Pull request merged**